salesforce/modules/bulk/client.bal

```diff
@@ -91,11 +91,11 @@ public isolated client class Client {
     # + extIdFieldName - Field name of the external ID incase of an Upsert operation
     # + return - returns job object or error
     @display {label: "Create Job"}
-    isolated remote function createJob(@display {label: "Operation"} Operation operation, 
-                                       @display {label: "SObject Name"} string sobj,
-                                       @display {label: "Content Type"} JobType contentType, 
-                                       @display {label: "External ID Field Name"} string extIdFieldName = "") 
-                                       returns @tainted @display {label: "Bulk Job"} error|BulkJob {
+    isolated remote function createJob(@display {label: "Operation"} Operation operation,
+                                        @display {label: "SObject Name"} string sobj,
+                                        @display {label: "Content Type"} JobType contentType,
+                                        @display {label: "External ID Field Name"} string extIdFieldName = "")
+                                        returns @display {label: "Bulk Job"} error|BulkJob {
```

Comment:
Is the tainted check ignored here from the original code?

Reply:
The Taint analyzer is disabled and the `--taint-check` option is removed from SLBeta1. So better we remove IMO. 
Refer the last point here. https://ballerina.io/downloads/swan-lake-release-notes/swan-lake-beta1/#breaking-changes